src/screens/UploadFiles.tsx

```diff
@@ -0,0 +1,193 @@
+import { ChangeEvent, useReducer, useState, useEffect } from "react";
+
+import ManifestDefinitions from "src/components/ManifestDefs";
+import { Button } from "../../src/components/Button";
+import { Alert } from "@us-gov-cdc/cdc-react";
+
+import * as tus from "tus-js-client";
+
+import { getEnv } from "../../src/utils/helperFunctions/env";
+
+interface FileUpload {
+  file: File;
+  manifest: string;
+}
+
+interface DispatchAction {
+  type: string;
+  value?: any;
+}
+
+function UploadFiles() {
+  const fileTypes = [".csv", ".hl7", ".txt"];
```

Comment:
File types can be any. We are not going to restrict the types of files a user can upload at this time.